courses/models.py

```diff
@@ -105,6 +111,10 @@ def live(self):
         """Returns a queryset of Course runs with live=True"""
         return self.get_queryset().live()
 
+    def is_available(self):
```

Comment:
Please rename to `available`. That would keep it in line with `live` and `with_text_id`.